samples/test/ssmlAddresses.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const fs = require('fs');
+const {assert} = require('chai');
+
+const ssmlAddresses = require('./../ssmlAddresses');
```

Comment:
Test needs to be refactored to use execSync and the commented out command in line 23 - you should be able to remove this line, and your test will invoke the function being tested like a user would invoke it on the command line (see https://github.com/ecrowdus/nodejs-text-to-speech/blob/updated-ssml/samples/test/listVoices.test.js#L21) as an example